### PR TITLE
chore: add docker builds

### PR DESCRIPTION
**Pull request by dj8yfo**
testing was performed on this [offshoot_testing_reproducible_testnet branch](https://github.com/dj8yfo/neardevhub-contract/tree/offshoot_testing_reproducible_testnet),

which differs from the branch of pr with this [commit](https://github.com/dj8yfo/neardevhub-contract/commit/dcd8cb4c8eccabda24a22da6441652663f2544ca) (relevant changes).

*offshoot_testing_reproducible_testnet* branch added `package.metadata.near.reproducible_build`
for 2 more contracts : `devhub-community` and `devhub-discussions`,
which was only needed for testing to confirm identical wasms were obtained
by running `cargo near build` procedure. these additional metadata sections
aren't needed in current pr, as these contracts aren't supposed to be built separately
(at least for deployments).

```bash
# confirmation, that wasms were obtained in a reproducible manner
# (nep330 metadata looks ok too)
a6144b70dd7692139b0e2b4e7be7bbb3a13aeda4966a6c6ce9277b8bb29afb61  community.devhub-test-3.testnet.wasm
5a50aecb76bb6d768791376e5ed9733dab28d1781e4e7827968d7c20f78579b1  devhub-test-3.testnet.wasm
be1a72acb6dfb54371bdeb8a3b1f81c6ab03b6b5f7649aa66d568765fef3eed3  discussions.gothamuniquehandle3.community.devhub-test-3.testnet.wasm
35d7ddea329134cad70e367c4f27e78e6539fbe145e4e669ff2fd72137ceb506  gothamuniquehandle3.community.devhub-test-3.testnet.wasm
35d7ddea329134cad70e367c4f27e78e6539fbe145e4e669ff2fd72137ceb506  standalone-devhub-community-3.testnet.wasm
be1a72acb6dfb54371bdeb8a3b1f81c6ab03b6b5f7649aa66d568765fef3eed3  standalone-devhub-discussions-3.testnet.wasm
```

### CODE REVIEW

**Comment by dj8yfo** [comment]
@race-of-sloths ping

**Comment by race-of-sloths** [comment]
@dj8yfo Thank you for your contribution! Your pull request is now a part of the [Race of Sloths](https://race-of-sloths.com)!
Congratulatory messages are spent out! Your results have exceeded our imagination!

[<picture>
    <source media="(prefers-color-scheme: dark)" srcset="https://badge.race-of-sloths.com/dj8yfo?type=bot&pr=NEAR-DevHub/neardevhub-contract/151&theme=dark">
    <source media="(prefers-color-scheme: light)" srcset="https://badge.race-of-sloths.com/dj8yfo?type=bot&pr=NEAR-DevHub/neardevhub-contract/151&theme=light">
    Shows profile picture for the author of the PR
</picture>](https://race-of-sloths.com/profile/dj8yfo)





<details><summary>Current status: <i>executed</i></summary>

| Reviewer | Score |
|--------|--------|
| @frol  | 5 |


Your contribution is much appreciated with a final score of 5!
You have received 53 (50 base + 5% lifetime bonus) Sloth points for this contribution


We would appreciate your feedback or contribution idea that you can submit [here](https://github.com/NEAR-DevHub/race-of-sloths/issues/new/choose)
</details><details><summary>What is the Race of Sloths</summary>
<p>
Race of Sloths is a friendly competition where you can participate in challenges and compete with other open-source contributors within your normal workflow

For contributors:

- Tag @race-of-sloths inside your pull requests
- Wait for the maintainer to review and score your pull request
- Check out your position in the [Leaderboard](https://race-of-sloths.com/leaderboard)
- Keep weekly and monthly streaks to reach higher positions
- Boast your contributions with a dynamic picture of your [Profile](https://race-of-sloths.com/profile/dj8yfo)

For maintainers:
- Score pull requests that participate in the Race of Sloths
- Engage contributors with fair scoring and fast responses so they keep their streaks
- Promote the Race to the point where the Race starts promoting you
- Grow the community of your contributors

Feel free to check [our website](https://race-of-sloths.com) for additional details!
</p>
</details> 

<details>
<summary>Bot commands</summary>

- For contributors
  - **Include a PR:** `@race-of-sloths include` to enter the Race with your PR
- For maintainers:
  - **Invite contributor** `@race-of-sloths invite` to invite the contributor to participate in a race or include it, if it's already a runner.
  - **Assign points:** `@race-of-sloths score [1/2/3/5/8/13]` to award points based on your assessment.
  - **Reject this PR:** `@race-of-sloths exclude` to send this PR back to the drawing board.
  - **Exclude repo:** `@race-of-sloths pause` to stop bot activity in this repo until `@race-of-sloths unpause` command is called

</details>

**Review by frol — APPROVED** [review]
@race-of-sloths score 5

**Pull request merged**
